entities/schema/schema.go

```diff
@@ -51,3 +53,35 @@ func Empty() Schema {
 func (s *Schema) SemanticSchemaFor() *models.Schema {
 	return s.Objects
 }
+
+func UpperCaseClassName(name string) string {
```

Comment:
I think that there's a word `Uppercase` so I think the method should be named `UppercaseClassName`